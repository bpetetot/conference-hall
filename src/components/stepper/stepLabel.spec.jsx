/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import StepLabel from './stepLabel'

describe('components/stepLabel', () => {
  it('should render', () => {
    const wrapper = shallow(<StepLabel label="boo" step={0} current={0} />)
    expect(wrapper).toMatchSnapshot()
  })
})
