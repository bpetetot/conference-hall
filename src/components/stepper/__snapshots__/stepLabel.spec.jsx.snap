// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/stepLabel should render 1`] = `
<div
  className="step-active-label"
>
  boo
</div>
`;
